src/shared/components/donate-definitions.ts

```diff
@@ -39,4 +39,4 @@ export const MEDIAN_DEV_MONTHLY_EUR = MEDIAN_DEV_SALARY / 12;
 export const FUNDED_DEVS = TOTAL_RECURRING_MONTHLY_EUR / MEDIAN_DEV_MONTHLY_EUR;
 
 //  Goal
-export const FUNDED_DEV_GOAL = 3;
+export const FUNDED_DEV_GOAL = 2;
```

Comment:
Removed this to be more realistic, and in any case we dont have any third dev for now.